docs/conf.py

```diff
@@ -23,8 +23,8 @@
 
 # General information
 project = "anndata"
-author = "AnnData development team"
-copyright = f"{datetime.now():%Y}, the AnnData development team."
+author = f"{project} developers"
+copyright = f"{datetime.now():%Y}, {author}"
```

Comment:
This looked a bit strange with the double period and using the class instead of the package/project name:
![image](https://user-images.githubusercontent.com/16916678/141360462-10116db3-66f1-4b8c-844e-bc199ac2e99e.png)